package.json

```diff
@@ -3,6 +3,7 @@
   "version": "2.1.0",
   "description": "Fantasy Land -compliant Either type",
   "license": "MIT",
+  "type": "module",
```

Comment:
For this change to work, we'd have convert the existing implementation .. are there other options?